src/graph_view.rs

```diff
@@ -147,6 +142,8 @@ impl<'a, N: Clone, E: Clone, Ty: EdgeType> GraphView<'a, N, E, Ty> {
             computed.comp_iter_bounds(n, &self.settings_style);
         });
 
+        computed.compute_selected_edges(self.g);
```

Comment:
I think here we should follow the same strategy as with nodes state computing: ether we iter explicitly and compute state for one node or iter in computation method (as you did). I beter like explicit iteration as it is done in node state computing.